### ISSUE TITLE
TypeError: Unconfigurable is not JSON serializable

### ISSUE THREAD

**Issue by ikajic** [nengo/nengo_gui#694]
Something is weird: when I create a new empty file in terminal (e.g. `touch tmp.py`) and try to open in GUI I get:
```
Traceback (most recent call last):
  File "/home/ivana/phd/workspace/nengo_gui/nengo_gui/swi.py", line 276, in handle_request
    text = getattr(self, command)(*args[1:], **db)
  File "/home/ivana/phd/workspace/nengo_gui/nengo_gui/server.py", line 87, in swi
    components = page.create_javascript()
  File "/home/ivana/phd/workspace/nengo_gui/nengo_gui/page.py", line 314, in create_javascript
    component_js = '\n'.join([c.javascript() for c in self.components])
  File "/home/ivana/phd/workspace/nengo_gui/nengo_gui/components/sim_control.py", line 181, in javascript
    js = self.javascript_config(info)
  File "/home/ivana/phd/workspace/nengo_gui/nengo_gui/components/component.py", line 100, in javascript_config
    return json.dumps(cfg)
  File "/usr/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: Unconfigurable is not JSON serializable
```

Also, I get the same error when resetting the layout? I am using nengo_gui installed from the master (`python setup.py develop`)

**Comment by ikajic** [comment]
Now I get the same error even when I just try to start the gui from terminal :/

**Comment by studywolf** [comment]
i also get this!

**Comment by ikajic** [comment]
Fixed via #695.

**Comment by tcstewar** [comment]
I just did an emergency merge into master to fix this:  https://github.com/nengo/nengo_gui/pull/695

What happened is that the recent changes to nengo.params.Parameter in nengo master changed the constructor, and so nengo_gui wasn't actually setting the default values any more.
